EYE_HEALTH_GUIDE.md

```diff
@@ -0,0 +1,127 @@
+
+### Importance of Regular Eye Exams & Corrective Lenses
+
+- **Comprehensive Eye Exams**: Have your eyes checked at least once every two years (more often if you have existing vision problems). Inform your eye‑care professional about your screen‑use habits.
+- **Computer‑Specific Eyewear**: Special “computer glasses” with an anti‑reflective coating and a prescription optimized for intermediate distance (about 50–70 cm) can reduce accommodative stress.
+
+## Long‑Term Eye Health Habits
+
+### Blinking Exercises & Artificial Tears
+
+- **Blinking Drill**: Set a timer to remind yourself to blink fully (upper and lower lids meeting) every 5–10 minutes.
+- **Artificial Tears**: Use preservative‑free lubricating eye drops if you experience persistent dryness. Apply them during breaks, not while staring at the screen.
+
+### Outdoor Time & Myopia Prevention
+
+- **Sunlight Exposure**: Spending at least 1–2 hours outdoors each day may help slow the progression of myopia (nearsightedness), especially in children and adolescents.
+- **Distance Viewing**: Make a habit of looking at distant objects (trees, buildings, horizon) whenever you step outside.
+
+### Nutrition & Hydration for Ocular Health
+
+- **Hydration**: Drink plenty of water throughout the day to support tear production.
+- **Nutrient‑Rich Foods**: Include foods high in omega‑3 fatty acids (salmon, flaxseeds), lutein and zeaxanthin (leafy greens, eggs), and vitamins A, C, E (carrots, citrus fruits, nuts) in your diet.
+
+## FAQs & Troubleshooting
+
+### Why do I still feel tired even with regular breaks?
+
+- **Insufficient Break Length**: Try extending your short breaks to 30–40 seconds.
+- **Unaddressed Dry Eyes**: Use artificial tears and practice conscious blinking.
+- **Underlying Vision Issues**: Schedule an eye exam to rule out uncorrected refractive errors or binocular vision problems.
+
+### How can I customize Safe Eyes for my individual needs?
+
+- **Settings Dialog**: Open the settings (`safeeyes -s`) to adjust break intervals, duration, and plugins.
+- **Configuration File**: Edit `~/.config/safeeyes/safeeyes.json` directly for advanced customization (e.g., adding custom break exercises, changing notification sounds).
+
+### Does Safe Eyes work with multiple monitors?
+
+Yes. Safe Eyes displays the break screen on all connected monitors by default. You can also configure it to show only on the primary monitor.
+
+### Can I temporarily disable Safe Eyes?
+
+Yes. Use the tray‑icon menu or run `safeeyes --disable` to pause breaks. Re‑enable with `safeeyes --enable` or via the tray menu.
+
+---
+
+*This guide is based on current clinical research and recommendations from eye‑care professionals. It is intended for informational purposes only and is not a substitute for professional medical advice. If you experience persistent or severe eye discomfort, consult an optometrist or ophthalmologist.*
```

Comment:
If we say that the guide is based on clinical research (which I am sure it is), we should also add some credible references